services/QuillLMS/app/queries/admin_diagnostic_reports/post_diagnostic_completed_view_query.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module AdminDiagnosticReports
+  class PostDiagnosticCompletedViewQuery < DiagnosticAggregateViewQuery
+    def root_query
+      <<-SQL
+        SELECT
+            diagnostic_id,
+            diagnostic_name,
+            aggregate_id,
+            name,
+            group_by,
+            post_students_completed,
+            AVG(growth_percentage) AS overall_skill_growth
+          FROM (#{super})
+          GROUP BY diagnostic_id, diagnostic_name, aggregate_id, name, group_by, post_students_completed
+      SQL
+    end
+
+    def specific_select_clause
+      <<-SQL
+        COUNT(DISTINCT performance.post_activity_session_id) AS post_students_completed,
+        /*
+        The value below is used to duplicate growth percentage calculations from teacher reports:
```

Comment:
Let's yank the comments outside of the HEREDOC. They're difficult to read and to differentiate from the SQL.